dot/node_test.go

```diff
@@ -95,7 +95,8 @@ func TestLoadGlobalNodeName(t *testing.T) {
 	}
 }
 
-//go:generate mockgen -destination=mock_service_registry_test.go -package=$GOPACKAGE github.com/ChainSafe/gossamer/lib/services Service,ServiceRegisterer
+//go:generate mockgen -destination=mock_service_registry_test.go -package=$GOPACKAGE github.com/ChainSafe/gossamer/lib/services Service
```

Comment:
add to mock_generate file?

Reply:
Annnndd it turns out `mock_service_registry_test.go` was unused, so removed that line and the mock file!

For the new go generate added, I'll move it to a `mocks_generate_test.go` in the next PR